hypertrace-trace-enricher/hypertrace-trace-enricher-impl/src/main/java/org/hypertrace/traceenricher/enrichment/clients/DefaultClientRegistry.java

```diff
@@ -142,4 +143,9 @@ public void shutdown() {
   protected ManagedChannel buildChannel(String host, int port) {
     return this.grpcChannelRegistry.forPlaintextAddress(host, port);
   }
+
+  protected ManagedChannel buildChannel(String host, int port, int maxMessageSize) {
```

Comment:
Later, we might need to provide some other config?

Should we add the method 
- `protected ManagedChannel buildChannel(String host, int port, int GrpcChannelConfig)`?

Reply:
yeah that will be better

Reply:
Currently the GrpcChannelConfig has just one field